### ISSUE TITLE
Add Jupyter Notebook Versions For Training Code

### ISSUE THREAD

**Issue by AlpOzgun** [CankayaUniversity/ceng-407-408-2025-2026-Multi-Label-Chest-X-Ray-Disease-Classification-and-Explainability#12]
The training code for the ai model is currently missing please upload the current Jupyter Notebook versions.

**Comment by BarkinV2** [comment]
Training scripts uploaded
